conductor/__beta__/job/worker.py

```diff
@@ -54,15 +59,10 @@ def _get_environment(self):
         self.environment['CONDUCTOR_DEADLINE_GROUP_NAME'] = self.deadline_group_name         
         self.environment['DCONFIG_ProxyUseSSL'] = str(self.deadline_use_ssl).lower()
         self.environment['CONDUCTOR_DEADLINE_CLIENT_VERSION'] = self.deadline_client_version
+        self.environment['DCONFIG_ProxyRoot'] = self.deadline_proxy_root
         
         if self.deadline_use_ssl:
             self.environment['DCONFIG_ProxySSLCertificate'] = self.deadline_ssl_certificate
-            self.environment['DCONFIG_ProxyRoot'] = "{};{}".format(self.deadline_proxy_root, self.deadline_ssl_certificate)

```

Comment:
Did you mean to remove the `;<deadline_ssl_certificate>` suffix?

Reply:
Yes - it appears to work without it.